src/openforms/registrations/contrib/objects_api/tests/test_api_endpoints.py

```diff
@@ -316,7 +309,17 @@ def test_retrieve_with_explicit_objects_api_group(self):
 
         data = response.json()
 
-        self.assertEqual(len(data), 3)
+        num_document_types = len(data)
+        # at least 6 distinct document types are defined in the fixture
+        self.assertGreaterEqual(num_document_types, 6)
```

Comment:
Is there a reason to use `assertGreaterEqual` instead of `assertEqual` here?

Reply:
future proofing in case more document types get added in the fixture and not having to update this exact assertion.

these tests are more tests that "anything" is returned and there may be more data (but rather we check that only unique descriptions are returned etc. instead of the exact count)